src/Robo/Commands/Setup/SettingsCommand.php

```diff
@@ -75,6 +89,14 @@ public function generateSiteConfigFiles() {
       $default_local_settings_file = "$multisite_dir/settings/default.local.settings.php";
       $project_local_settings_file = "$multisite_dir/settings/local.settings.php";
 
+      // Generate default.includes.settings.php.
+      $blt_includes_settings_file = $this->getConfigValue('blt.root') . '/settings/default.includes.settings.php';
+      $default_includes_settings_file = "$multisite_dir/settings/default.includes.settings.php";
+
+      // Generate sites/settings/global.settings.default.php.
+      $blt_glob_settings_file = $this->getConfigValue('blt.root') . '/settings/global.settings.default.php';
+      $default_glob_settings_file = $this->getConfigValue('docroot') . "/sites/settings/global.settings.default.php";
```

Comment:
This is part of a loop that runs once for every multisite, and yet it doesn't do anything multisite-specific. So it's just copying the same file to the same place over and over. I'm not sure if that's bad or not, just odd.